CompositionRootBuilder/CompositionRootBuilder.csproj

```diff
@@ -0,0 +1,18 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.1</TargetFramework>
+    <AssemblyName>Messerli.CompositionRootBuilder</AssemblyName>
+    <RootNamespace>Messerli.CompositionRootBuilder</RootNamespace>
+    <LangVersion>8.0</LangVersion>
+    <Nullable>enable</Nullable>
+    <NullableContextOptions>enable</NullableContextOptions>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <WarningsAsErrors />
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+  </PropertyGroup>
+  <ItemGroup>
+    <AdditionalFiles Include="readme.md" />
+  </ItemGroup>
+  <Import Project="..\.paket\Paket.Restore.targets" />
+</Project>
```

Comment:
Add a newline

Reply:
Addressed in: https://github.com/messerli-informatik-ag/composition-root-builder/commit/bd2723820a7a9ff16b02363ce62cea4d9e3e768c